hermes/src/main.rs

```diff
@@ -51,36 +53,47 @@ fn main() {
                     .help("the relative or absolute directory of the networks to server")
                     .required(false)
                 )
-        )
-        .subcommand(
-            SubCommand::with_name("start")
-                .about("start explorer server")
+                .arg(Arg::with_name("TEMPLATE")
+                    .long("template")
+                    .takes_value(true)
+                    .value_name("TEMPLATE")
+                    .help("either 'mainnet' or 'testnet'; may be given multiple times")
+                    .required(false)
+                    .multiple(true)
+                    .default_value("mainnet")
```

Comment:
add `possible_values` too:

```rust
  .arg(Arg::with_name("TEMPLATE")
         // ...
        .possible_values(&["mainnet", "testnet"])
        .default_value("mainnet")
```

that way it is `clap` which is going to handle the error of unknown template input and will provide a more meaningful error message